test/config.js

```diff
@@ -31,7 +31,7 @@ describe('@config@', function () {
   });
 
   it('should install provided @selenium.version@', function (done) {
-    var ver = '^2.53.1';
+    var ver = '2.53.3';
```

Comment:
Were the tests failing here?  I'm fine w/ this change, but perhaps at some point down the line we need to clean up this test.